CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 1.2.1 (2023-04-19)
```

Comment:
Our convention is to only add a date when we do a release. Often we won't do a release when we merge a PR unless it needs to go out immediately for some reason.

Also, you can see in this case that 1.2.0 hasn't been released yet, so we can list the new change under that version.